client/components/popover/test/index.test.js

```diff
@@ -34,14 +34,14 @@ describe( 'Popover', () => {
 			screen.queryByText( 'Popover Content' )
 		).not.toBeInTheDocument();
 
-		act( () => {
-			userEvent.click( screen.getByTestId( 'base-component' ) );
+		act( async () => {
+			await userEvent.click( screen.getByTestId( 'base-component' ) );
 		} );
 
 		expect( screen.queryByText( 'Popover Content' ) ).toBeInTheDocument();
 
-		act( () => {
-			userEvent.click( screen.getByTestId( 'base-component' ) );
+		act( async () => {
```

Comment:
```suggestion
		await act( async () => {
```

Reply:
Adding `await` here fixes the following console error.

<img width="1461" height="198" alt="Screenshot 2025-08-25 at 2 24 19 PM" src="https://github.com/user-attachments/assets/08cf6e2b-061c-4b95-9da7-86857940c983" />